packages/cli/src/deployer/types/types.ts

```diff
@@ -51,7 +52,8 @@ export class TypesDeployer extends BaseTypesDeployer {
         return Promise.resolve();
     }
     fileFilter(file: string): boolean {
-        return file.indexOf('types/') > -1
+        return file.split(path.sep).reverse().find(pathElement => 
+            pathElement === "types" || pathElement === "interfaces") === 'types'
```

Comment:
If this function is the same for types and interfaces deployers we can export it in a common place.

Reply:
Its not the same, the comparison at the end is different.